unlock_indicator.c

```diff
@@ -26,10 +26,12 @@
 #include <time.h>
 
 extern double circle_radius;
+extern double ring_width;
 
 #define BUTTON_RADIUS (circle_radius)
-#define BUTTON_SPACE (BUTTON_RADIUS + 5)
-#define BUTTON_CENTER (BUTTON_RADIUS + 5)
+#define RING_WIDTH (ring_width)
+#define BUTTON_SPACE (BUTTON_RADIUS + RING_WIDTH)
```

Comment:
If we're adding 5 before, shouldn't we be doing something like `BUTTON_RADIUS + RING_WIDTH - 2`? I'll test this tonight or tomorrow (my main rig currently has a dead CPU/motherboard), but this seems a but wrong at a glance.

Reply:
We are both wrong :laughing: . The ring is drawn by cairo with the brush centered on `BUTTON_RADIUS`.  So really it should be `BUTTON_SPACE (BUTTON_RADIUS + (RING_WIDTH / 2))`. The reason why 5 worked before is because 5 > 7 / 2. I'll fix that shortly.

I'm also working on cleaning up the options and creating an actual command line help but no promises. This might be something I should push to i3-lock first, then add i3-lock-color options after that's merged in...

Reply:
Ahh, lol. 

And alright! I've been meaning to clean up i3lock options stuff for a while (a config file is looking more appealing as options keep getting added), but I've never really been able to decide on something that upstream would probably want.

Reply:
Are they particular about that issue?  I don't want to work on it if they are going to deny it outright. In other news, I think I'm just going to stick to the ring width on this branch and work on the help in a different branch.